### PR TITLE
[CABMAN] Fix a problem with creating temp files on some Windows system

### PR DESCRIPTION
**Pull request by tkreuzer**
See [CORE-14431](https://jira.reactos.org/browse/CORE-14431)

### CODE REVIEW

File: sdk/tools/cabman/CCFDATAStorage.cxx [hunk 1]
```diff
@@ -51,12 +51,17 @@ CCFDATAStorage::~CCFDATAStorage()
...
 ULONG CCFDATAStorage::Create()
 {
     if ((FileHandle = tmpfile()) == NULL)
-        return CAB_STATUS_CANNOT_CREATE;
+    {
+        /* Workaround for breakage on some Windows system */
+        FileHandle = fopen(tmpnam(NULL) + 1, "wb");
+        if (FileHandle == NULL)
+            return CAB_STATUS_CANNOT_CREATE;
+    }
```
Comment (HBelusca) [inline sdk/tools/cabman/CCFDATAStorage.cxx]: Do we really know what causes this breakage?

Comment (learn-more) [reply sdk/tools/cabman/CCFDATAStorage.cxx]: first `tmpfile()` fails.

Comment (tkreuzer) [reply sdk/tools/cabman/CCFDATAStorage.cxx]: For some reason tmpfile() fails. Only happens with GCC builds, only on some Windows 10 systems. This fixes it for me. Everything else is left as an exersize to the reader ;-)

**Comment by tkreuzer** [comment]
Hmm. Looks like this PR is only blocked by ... @ColinFinck :)
I could now argue that all I claimed is that this fixes build for me, and whoever can't reproduce it, might just not have put enough effort into it, and whoever finds things hard to believe is free to call me a liar, but luckily it seems like MS has fixed the issue with the latest VS update, so this makes this PR obsolete.

**Comment by HBelusca** [comment]
Interesting! Diffing old VS crt vs new VS crt one could help knowing what happened there :smiley:

**Comment by tkreuzer** [comment]
Reopening, since it is still not fixed for me, not sure how and why it succeeded in between.

**Pull request merged**